rali/rali/include/bounding_box_graph.h

```diff
@@ -29,5 +29,6 @@ class BoundingBoxGraph : public MetaDataGraph
 public:
     void process(MetaDataBatch* meta_data) override;
     void update_meta_data(MetaDataBatch* meta_data, decoded_image_info decode_image_info) override;
+    void update_random_bbox_meta_data(CropCordBatch* _random_bbox_crop_cords_data, MetaDataBatch* meta_data, decoded_image_info decoded_image_info) override;
```

Comment:
It is better to keep the existing naming convention for consistency.
For e.g _xxx is used for member variables of a class. Please remove _random_bbox_crop_cords_data to random_bbox_crop_cords_data

Reply:
resolved